### ISSUE TITLE
When the collection view is loaded for the first time, the animation does not work.

### ISSUE THREAD

**Issue by jangsy7883** [marcosgriselli/ViewAnimator#23]
## Actual Behavior
When the collection view is loaded for the first time, the animation does not work.

## Steps to Reproduce the Problem
        sender.isEnabled = false
        activityIndicator.stopAnimating()
        items = Array(repeating: nil, count: 20)
        self.collectionView!.reloadData()
        self.collectionView!.animateViews(animations: [AnimationType.from(direction: .right, offset: 30.0)],
                                          completion: {
                                            sender.isEnabled = true
        })

**Comment by marcosgriselli** [comment]
I've implemented a solution on 7fb4da4e3457f38fb885cc2f38765e512dfee5f4 as suggested in StackOverflow: [link](https://stackoverflow.com/a/30668519/2892512) 

```swift 
collectionView?.reloadData()
collectionView?.performBatchUpdates({
    self.collectionView?.animateViews(animations: animations,
                                      completion: {
                                        sender.isEnabled = true
    })
}, completion: nil)
```

I'm not 100% convinced with the solution as some users mentioned that it didn't work with them. I'll investigate more so if any expert in UICollectionView data reloading and status reads this please reach out to me. 
Here's a simple animation result: 

![collectionanimation](https://user-images.githubusercontent.com/14804033/36705031-e58c6c9e-1b41-11e8-81a1-d6e04fcf5739.gif)

**Comment by marcosgriselli** [comment]
@loadedjd I just pushed an example of animating on push. Not the cleanest one but it should work: 9ac0e10. Just remember to set your data array to be initially empty.
